_opcodes/op134-bgee.html

```diff
@@ -0,0 +1,34 @@
+---
+n: 134
+name: "State: Petrification"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the petrification effect to the targeted creature(s). This effects kills & removes PC's from the party, and ends the game if used on the Protagonist.
```

Comment:
can be merged back, but make sure to include the iwd/pst exception.

Reply:
Done.